Build is broken for Avalonia.Android

**Describe the bug**
Build for Avalonia.Android fails:
16>------ Build started: Project: Avalonia.Android, Configuration: Debug Any CPU ------
16>D:\src\avalonia\src\Android\Avalonia.Android\AvaloniaView.cs(18,28,18,29): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
16>D:\src\avalonia\src\Android\Avalonia.Android\Platform\SkiaPlatform\AndroidFramebuffer.cs(35,19,35,47): error CS0019: Operator '*' cannot be applied to operands of type 'double' and 'Vector'
16>Done building project "Avalonia.Android.csproj" -- FAILED.

**To Reproduce**
Build Avalonia.Android.csproj

**Expected behavior**
Should be built without failure.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**

- OS: Windows 10
- Version [master head]

**Additional context**

--- a/src/Avalonia.Visuals/Vector.cs
+++ b/src/Avalonia.Visuals/Vector.cs
@@ -86,6 +86,15 @@ namespace Avalonia
         /// Scales a vector.
         /// </summary>
         /// <param name="vector">The vector.</param>
+        /// <param name="scale">The scaling factor.</param>
+        /// <returns>The scaled vector.</returns>
+        public static Vector operator *(double scale, Vector vector)
+            => Multiply(vector, scale);
+
+        /// <summary>
+        /// Scales a vector.
+        /// </summary>
+        /// <param name="vector">The vector.</param>
         /// <param name="scale">The divisor.</param>
         /// <returns>The scaled vector.</returns>
         public static Vector operator /(Vector vector, double scale)

--- a/tests/Avalonia.Visuals.UnitTests/VectorTests.cs
+++ b/tests/Avalonia.Visuals.UnitTests/VectorTests.cs
@@ -105,5 +105,15 @@ namespace Avalonia.Visuals.UnitTests
 
             Assert.Equal(expected, Vector.Multiply(vector, 2));
         }
+
+        [Fact]
+        public void Scale_Vector_Should_Be_Commutative()
+        {
+            var vector = new Vector(10, 2);
+
+            var expected = vector * 2;
+
+            Assert.Equal(expected, 2 * vector);
+        }
     }
 }

